src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/ClientWebSocketOptions.cs

```diff
@@ -12,6 +13,8 @@ public sealed class ClientWebSocketOptions
     {
         private bool _isReadOnly; // After ConnectAsync is called the options cannot be modified.
         private List<string>? _requestedSubProtocols;
+        private Version _version = Net.HttpVersion.Version11;
```

Comment:
Yes, because it has collision with the property's name https://github.com/dotnet/runtime/blob/8ef8b4c6263ccbc189fb608f80f22acc9dd6d346/src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/ClientWebSocketOptions.cs#L38